src/test/mir-opt/const_prop/tuple_literal_propagation.main.ConstProp.diff

```diff
@@ -2,40 +2,40 @@
+      let _2: ();                          // in scope 0 at $DIR/tuple_literal_propagation.rs:6:5: 6:15
+      let mut _3: (u32, u32);              // in scope 0 at $DIR/tuple_literal_propagation.rs:6:13: 6:14
       scope 1 {
-          debug x => _1;                   // in scope 1 at $DIR/tuple_literal_propagation.rs:3:9: 3:10
+          debug x => _1;                   // in scope 1 at $DIR/tuple_literal_propagation.rs:4:9: 4:10
       }
   
       bb0: {
-          StorageLive(_1);                 // scope 0 at $DIR/tuple_literal_propagation.rs:3:9: 3:10
-          (_1.0: u32) = const 1_u32;       // scope 0 at $DIR/tuple_literal_propagation.rs:3:13: 3:19
-          (_1.1: u32) = const 2_u32;       // scope 0 at $DIR/tuple_literal_propagation.rs:3:13: 3:19
-          StorageLive(_2);                 // scope 1 at $DIR/tuple_literal_propagation.rs:5:5: 5:15
-          StorageLive(_3);                 // scope 1 at $DIR/tuple_literal_propagation.rs:5:13: 5:14
--         _3 = _1;                         // scope 1 at $DIR/tuple_literal_propagation.rs:5:13: 5:14
-+         _3 = const (1_u32, 2_u32);       // scope 1 at $DIR/tuple_literal_propagation.rs:5:13: 5:14
+          StorageLive(_1);                 // scope 0 at $DIR/tuple_literal_propagation.rs:4:9: 4:10
+          (_1.0: u32) = const 1_u32;       // scope 0 at $DIR/tuple_literal_propagation.rs:4:13: 4:19
+          (_1.1: u32) = const 2_u32;       // scope 0 at $DIR/tuple_literal_propagation.rs:4:13: 4:19
+          StorageLive(_2);                 // scope 1 at $DIR/tuple_literal_propagation.rs:6:5: 6:15
+          StorageLive(_3);                 // scope 1 at $DIR/tuple_literal_propagation.rs:6:13: 6:14
+-         _3 = _1;                         // scope 1 at $DIR/tuple_literal_propagation.rs:6:13: 6:14
++         _3 = const (1_u32, 2_u32);       // scope 1 at $DIR/tuple_literal_propagation.rs:6:13: 6:14
 +                                          // ty::Const
 +                                          // + ty: (u32, u32)
 +                                          // + val: Value(ByRef { alloc: Allocation { bytes: [1, 0, 0, 0, 2, 0, 0, 0], relocations: Relocations(SortedMap { data: [] }), init_mask: InitMask { blocks: [255], len: Size { raw: 8 } }, size: Size { raw: 8 }, align: Align { pow2: 2 }, mutability: Not, extra: () }, offset: Size { raw: 0 } })
 +                                          // mir::Constant
-+                                          // + span: $DIR/tuple_literal_propagation.rs:5:13: 5:14
++                                          // + span: $DIR/tuple_literal_propagation.rs:6:13: 6:14
 +                                          // + literal: Const { ty: (u32, u32), val: Value(ByRef { alloc: Allocation { bytes: [1, 0, 0, 0, 2, 0, 0, 0], relocations: Relocations(SortedMap { data: [] }), init_mask: InitMask { blocks: [255], len: Size { raw: 8 } }, size: Size { raw: 8 }, align: Align { pow2: 2 }, mutability: Not, extra: () }, offset: Size { raw: 0 } }) }
```

Comment:
This seems to be a "significant" diff -- i.e., not just line numbers changing. Maybe a bug in how github is showing it or a rebase error?

Reply:
This is just line numbers changing AFAICT. Diffs of diffs are hard to parse for me also.

Reply:
Things like these can be avoided by adding new comment lines in existing empty lines or just adding them at the end of the file. We have no policy on this though

Reply:
Yes this is just line numbers changing - the existing file didn't have an empty line to fit a comment into, and I thought putting it at the end of the file would be inconsistent with everything else.